src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -92,162 +96,325 @@ fn default_endpoint() -> String {
 
 impl_generate_config_from_default!(StackdriverConfig);
 
-#[async_trait::async_trait]
-impl SinkConfig for StackdriverConfig {
-    async fn build(&self, cx: SinkContext) -> crate::Result<(VectorSink, Healthcheck)> {
-        let auth = self.auth.build(Scope::MonitoringWrite).await?;
+const GCP_MAX_BATCH_SIZE: usize = 200;
```

Comment:
Done

Reply:
I'm not seeing anything in that link that explains why the batch size needs to be < 200 or why the call frequency needs to be 10 seconds. Can you explain the reasoning?

Reply:
The link points to that table showcasing the 200 time series size limit and the 5 secs minimum frequency rate for a point. I picked 10secs because when we initially used that code in the PR in my company, we found out it to be too tight (GCP complained a couple of times a day about reaching that time limit). That was about a year ago. Maybe now 5 secs would work.

![gcp_value](https://user-images.githubusercontent.com/144545/220371070-74bcdadb-a121-4d41-b9b0-3df2d2e2af24.jpeg)